iphone/Maps/UI/Settings/Cells/SettingsTableViewLinkCell.swift

```diff
@@ -1,19 +1,22 @@
 @objc
 final class SettingsTableViewLinkCell: MWMTableViewCell {
-  @IBOutlet fileprivate weak var title: UILabel!
-  @IBOutlet fileprivate weak var info: UILabel?
 
   override func awakeFromNib() {
     super.awakeFromNib()
-    self.styleName = "Background"
-    self.title.styleName = "regular17:blackPrimaryText"
-    self.info?.styleName = "regular17:blackSecondaryText"
+    setupCell()
   }
 
-  @objc func config(title: String, info: String?) {
-    self.title.text = title
+  private func setupCell() {
+    styleName = "Background"
+    textLabel?.styleName = "regular17:blackPrimaryText"
+    textLabel?.numberOfLines = 0
+    textLabel?.lineBreakMode = .byWordWrapping
```

Comment:
Is there any overhead in the generated code with repeated optional access? Won't this code work faster?
```suggestion
    if let textLabel {
      textLabel.styleName = "regular17:blackPrimaryText"
      textLabel.numberOfLines = 0
      textLabel.lineBreakMode = .byWordWrapping
    }
```

Reply:
Do we ever expect textLabel to be nil here and in other similar places? Shouldn't we crash/assert immediately if it's nil?